packages/framework/ai-collab/src/test/explicit-strategy/utils.spec.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+// eslint-disable-next-line import/no-internal-modules
+import { findClosestStringMatch, levenshteinDistance } from "../../explicit-strategy/utils.js";
+
+describe("levenshteinDistance", () => {
+	it("should return 0 for identical strings", () => {
+		const distance: number = levenshteinDistance("same", "same");
+		assert.strictEqual(distance, 0);
+	});
+
+	it("should return the length of the second string if the first is empty", () => {
+		const distance = levenshteinDistance("", "test");
+		assert.strictEqual(distance, 4);
+	});
+
+	it("should return the length of the first string if the second is empty", () => {
+		const distance = levenshteinDistance("test", "");
+		assert.strictEqual(distance, 4);
+	});
+
+	it("should handle simple single-character difference", () => {
+		const distance = levenshteinDistance("cat", "car");
+		// "cat" -> "car" (1 substitution)
+		assert.strictEqual(distance, 1);
+	});
+
+	it("should handle insertion and deletion differences", () => {
+		// "kitten" -> "sitting"
+		// One substitution (k->s), one substitution (e->i), one insertion (g), total 3 edits
+		const distance = levenshteinDistance("kitten", "sitting");
+		assert.strictEqual(distance, 3);
+	});
+
+	it("should handle more complex cases", () => {
+		// "gumbo" -> "gambol"
+		// "g" matches "g"
+		// "u" -> "a" substitution (1)
+		// "m" matches "m"
+		// "b" -> "b" matches
+		// "o" -> "o" matches
+		// then "l" insertion (2 edits total)
+		const distance = levenshteinDistance("gumbo", "gambol");
+		assert.strictEqual(distance, 2);
+	});
+});
+
+describe("findClosestStringMatch", () => {
+	it("should return the exact match if it exists", () => {
+		const matches = ["cat", "car", "cut", "coat"];
+		const best = findClosestStringMatch("car", matches);
+		assert.strictEqual(best, "car");
+	});
+
+	it("should return the single closest match if no exact match exists", () => {
+		const matches = ["cat", "car", "cut", "coat"];
+		const best = findClosestStringMatch("cart", matches);
+		// "cart" vs "cat": distance = 1 (insert 'r')
+		// "cart" vs "car": distance = 1 (insert 't')
+		// "cart" vs "cut": distance = 2
+		// "cart" vs "coat": distance = 2
+		// If there's a tie, which one you get depends on your iteration order;
+		// for example, "cat" might be found first. Adjust test as needed.
+		assert.strictEqual(matches.includes(best), true);
```

Comment:
This is not a great condition for the test, this would pass even if `best` is not the actual best match, like (in this example) `coat`. We should make this check against an explicit value since we know the order in which we're specifying the candidates.